backends/eopkg/eopkgBackend.py

```
@@ -604,25 +661,33 @@ def get_updates(self, filters):
-            pkg, repo = self.packagedb.get_package_repo(package, None)
+            try:
+                pkg, data = self._get_package_obj_from_id(package_id)
+            except PkError:
+                continue
+
+            if pkg.name not in upgradables:
+                continue
+
             version = self.__get_package_version(pkg)
-            id = self.get_package_id(pkg.name, version, pkg.architecture, repo)
+            id = self.get_package_id(pkg.name, version, pkg.architecture, data)
```

Ruff (0.15.12)

[error] Variable `id` is shadowing a Python builtin

(A001)